### ISSUE TITLE
What happen to 'toDate' parameter in didselect date method in Delegate?

### ISSUE THREAD

**Issue by stshelton** [shoheiyokoyama/Koyomi#23]
Having trouble getting end date when using the selection mode "sequence".  I had a previous version of this library on a different project and the "didSelect date" method in delegate had a "toDate" parameter that would provide the last date in sequence. Currently, that delegate method only has "didSelect date" which only returns the first date in the sequence even after the user clicked another date to complete sequence. Am I missing another function in the delegate, if not what would be the best method of retrieving the end date of the sequence of dates?

Also, I had an issue with Sat column disappearing when collection view is the same size of the screen. Once I added a trailing and leading constraint of 5 Sat column came back

**Comment by v1ntage** [comment]
I'm having these exact two issues too.

- Sat disappears at bounds.width unless you make it eg. width + 1 and then it's fine.
- Swift 2.3 version at least: the example is showing the wrong date when a user selects them. Eg. Feb 7 shows as Feb 6 when print in console.
- As described by stshelton, Sequence only shows the first one. Not only this, but if you select a sequence of dates going forwards, both selects are the same date.
Eg. Feb 2 (First) and then Feb 7 (second) show as Feb 2 on both clicks. But if you select it the other way, Feb 7 (first) and then Feb 2 (second) then the two selects are different. Still suffers from the above wrong date issue though.

Overall these issues make it unusable in its current state, unfortunately, as this is a great lightweight solution.

**Comment by ivanl001** [comment]
Same issues with the two guys before, the end selected time is not right

**Comment by danparnella** [comment]
I would agree that the "toDate" parameter should be included again in the "didSelect" method, but as a workaround for now you can use the "shouldSelectDates" method which does include both "date" and "toDate".
